ui/app/pages/create-account/connect-custodial/trustvault/email/email-screen.js

```diff
@@ -0,0 +1,160 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Button from '../../../../../components/ui/button'
+import TextField from '../../../../../components/ui/text-field'
+import { CONNECT_HARDWARE_ROUTE } from '../../../../../helpers/constants/routes'
+
+const ENTER_KEY = 13
+export class EmailScreen extends PureComponent {
+  state = {
+    email: null,
+  }
+
+  renderBackButton () {
+    return (
+      <div className="sw-connect__back" onClick={(_) => this.props.history.push(CONNECT_HARDWARE_ROUTE)}>
+        <div className="sw-connect__list__back-caret" />
+        <div className="sw-connect__list__back-caret__back">{this.context.t('back')}</div>
+      </div>
+    )
+  }
+
+  renderNextButton () {
+    return (
+      <Button
+        className="sw-connect__connect-btn"
+        type="primary"
+        onClick={(_) => this.props.getTrustVaultPinChallenge(this.state.email)}
+        disabled={!this.state.email}
+      >
+        {this.context.t('next')}
+      </Button>
+    )
+  }
+
+  renderEmailInputBox () {
+    return (
+      <div className="sw-connect__email-field">
+        <TextField
+          autoFocus
+          type="text"
+          placeholder="Email address"
+          largeLabel
+          fullWidth
+          onChange={(event) => this.setState({ email: event.target.value })}
+          onKeyDown={(event) => {
+            if (event.keyCode === ENTER_KEY) {
+              this.props.getTrustVaultPinChallenge(this.state.email)
+            }
+          }}
+        />
+      </div>
+    )
+  }
+  renderTrustVaultInfoBox () {
+    return (
+      <div className="sw-connect__info-box">
+        <img className="sw-connect__info-box__info-icon" src="images/tvInfo.png" />
+        <div className="sw-connect__info-box__not-user">{this.context.t('trustVaultNotUser')}</div>
+        <div className="sw-connect__info-box__ios">{this.context.t('trustVaultIos')}</div>
```

Comment:
This seems like it will not last very long, since the android app will come out and you can sign up via email. This can probably be omitted along with the associated locale message.